mcstatus/server.py

```diff
@@ -13,6 +15,17 @@
 __all__ = ["MinecraftServer", "MinecraftBedrockServer"]
 
 
+def ensure_valid_ip(host: object, port: object):
+    if not isinstance(host, str):
+        raise TypeError(f"Host must be a string address, got {type(host)} ({host!r})")
+    if not isinstance(port, int):
+        raise TypeError(f"Port must be an integer port number, got {type(port)} ({port})")
+    if port >= 36635 or port < 0:
```

Comment:
```suggestion
    if port > 65535 or port < 0:
```
2**16 is 65536 and 65535 is the maximum for TCP ports.